ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/disconnected/private/sts/custom-endpoints/cucushift-installer-rehearse-aws-ipi-disconnected-private-sts-custom-endpoints-workflow.yaml

```diff
@@ -0,0 +1,9 @@
+workflow:
+  as: cucushift-installer-rehearse-aws-ipi-disconnected-private-sts-custom-endpoints
```

Comment:
If "custom-endpoints" is a must for sts + disconnected clusters, maybe we can delete custom-endpoints subfolder from the path?

Reply:
Per the previous discussions, we want to add two workflows:
1. Pure disconnect-sts cluster to cover general configuration.
2. Disconnect-sts cluster + some specific EPs to cover customer cases.

Reply:
get it.